packages/backend/server/src/core/user/resolver.ts

```diff
@@ -70,6 +71,18 @@ export class UserResolver {
     };
   }
 
+  @Query(() => [PublicUserType], {
+    name: 'usersByIds',
+    description: 'Find users by ids',
+  })
+  async findUsersByIds(
```

Comment:
It needs to be a publicly accessible interface, allowing anonymous access when sharing documents. Strong request rate limiting protection needs to be added.